Bootloader/Adapters/Src/flash_adapter.c

```diff
@@ -354,7 +354,7 @@ FlashAdapter_program(uint32_t address, uint8_t* buffer, uint32_t length) {
 
     if ((length / flash_word) != 0 ) {
 
-        for (uint32_t i = 0; i < length / flash_word; i++) {
+        for (uint32_t i = 0; i < length / flash_word; ++i) {

```

Comment:
0U

Reply:
Fixed. It would have more sense to have some tool like clang-tidy or something. Checking this manually is a pain in the ass.